### PR TITLE
Add new Politician profession

### PR DESCRIPTION
**Pull request by Kanexan**
#### Summary

`SUMMARY: Features "This adds a new starting profession to the game, Career Politician"`
#### Purpose of change
This change adds a new starting profession which starts with the Skilled Liar trait plus Bartering 4 and Speaking 6, as well as a suit and tie, dress shoes, a tieclip, a set of gender-variable underwear, a briefcase, a UPS cell phone, one file, a cigar, a gold watch, and a refillable lighter. It costs 4 points.

### CODE REVIEW

File: data/json/professions.json [hunk 1]
```diff
@@ -3709,5 +3709,19 @@
...
       "female": [ "chestwrap" ]
     }
-  }
-]
+  },
+  {
+    "type": "profession",
+    "ident": "politician",
+    "name": { "male": "Career Politician", "female": "Career Politician" },
+    "description" : "You've spent your life appealing to the people, persuading many and promising much throughout your time in office. Now that your voting base wants to eat you alive, winning hearts and minds just got that much harder.",
```
Comment (Rail-Runner) [inline data/json/professions.json]:
```suggestion
    "description" : "You've spent your life appealing to the people, persuading many and promising much throughout your time in office.  Now that your voting base wants to eat you alive, winning hearts and minds just got that much harder.",
```

File: data/json/professions.json [hunk 2]
```diff
@@ -3709,5 +3709,19 @@
       },
       "female": [ "chestwrap" ]
     }
-  }
-]
+  },
+  {
+    "type": "profession",
+    "ident": "politician",
+    "name": { "male": "Career Politician", "female": "Career Politician" },
```
Comment (Rail-Runner) [inline data/json/professions.json]:
```suggestion
    "name": "Career Politician",
```

**Comment by Night-Pryanik** [comment]
This is almost a copy of a `Lawyer` profession, with one more point in both barter and speech and slightly better starting items. If this profession is going to get in the game, at least increase its cost.

Honestly I don't see much need for this profession, if only for roleplay reasons.

**Comment by Kanexan** [comment]
That's fair; I thought that aside from the roleplaying reasons, the bartering and speech might become useful when dealing with NPCs, once they've been more fleshed out. I'll come back and try to rebalance it in the morning.

**Comment by Rail-Runner** [comment]
Well, in that case it might get higher skill levels along with higher cost.

**Comment by DemAvalon** [comment]
it needs to have skilled liar as its profession trait.

**Comment by jeremyshannon** [comment]
Yeah, the joke goes that a lawyer is the larval form of a politician. This should definitely be a higher skilled, better equipped, (+ skilled liar) more costly version of lawyer, like bionic prepper is to prepper. Now that factions are becoming real, having a profession designed to schmooze NPCs into allies from day one makes sense.

**Comment by Ilysen** [comment]
@DemAvalon I came into this pull request hoping to see a comment like this. Posthaste!

**Comment by Kanexan** [comment]
Alright, I've updated this profession; it's now both more expensive and slightly better equipped, as well as significantly more skilled.
Plus the Career Politician now has Skilled Liar, a glaring omission that has now been rectified.

**Comment by LaVeyanFiend** [comment]
Needs a lint: http://dev.narc.ro/cataclysm/format.html

**Comment by davidpwbrown** [comment]
You need to modify the summary line in your first post to match the template.
`SUMMARY: Balance "Player's weight varies with BMI and size"`
for example ^

**Comment by Kanexan** [comment]
jenkins rebuild

**Comment by Kanexan** [comment]
jenkins rebuild

**Comment by LaVeyanFiend** [comment]
Lint again

**Comment by LaVeyanFiend** [comment]
You're not supposed to remove the extra line when you lint.

File: data/json/professions.json [hunk 3]
```diff
@@ -3709,5 +3709,35 @@
...
+      "skinny_tie",
+      "file",
+      "ref_lighter",
+      "cigar"
+    ],
+    "male": [ "boxer_shorts" ],
+    "female": [ "bra", "panties" ]
   }
-]
+}
```
Comment (LaVeyanFiend) [inline data/json/professions.json]: Don't remove the empty line after this when you lint, that's why it shows the red error sign.

Comment (Kanexan) [reply data/json/professions.json]: So put in one empty line—3744, which will be blank—and that will fix the issue?

Comment (LaVeyanFiend) [reply data/json/professions.json]: Yes, that will fix it. It threw me for a loop on my first PR too

Comment (LaVeyanFiend) [reply data/json/professions.json]: Wrong bracket at the end, needs to be square( ] ) rather than curly ( } ) for the end of a .json

Comment (Kanexan) [reply data/json/professions.json]: Alright. I _think_ that this should solve everything. I sure hope it does, anyways.

**Pull request merged**